apps/avatax/scripts/migrations/run-webhooks-migration.ts

```diff
@@ -8,6 +7,9 @@ const args = process.argv.slice(2);
 const dryRun = args.includes("--dry-run");
 
 const runMigration = async () => {
+  // Must use dynamic import for env variables to load properly
+  const { fetchCloudAplEnvs, verifyRequiredEnvs } = await import("./migration-utils");
```

Comment:
to avoid that in the future we can

1. `require('dotenv').config()` in the first line (and disable eslint sort lines)
2. run it in the script `node -r dotenv/config your_script.js`

Reply:
@krzysztofzuraw